### ISSUE TITLE
Copying lines from "Filter window" is trunkated

### ISSUE THREAD

**Issue by joko654** [zarunbal/LogExpert#30]
When copying a filtered line from the "Filter window" it is trunkated to the text visible in the "Filter window" and therefore has 4678 characters including trunation marker "..." excluding the line number in front of it.

My expectation: When copying to clip board the whole line is copied and not just the visible part.

Thanks.

**Comment by zarunbal** [comment]
Identified the Problem, the clipboard handling if you press strg+c is handled by the datagrid and therefore only the visible content is copied. (Note for me)

**Comment by Hirogen** [comment]
somehow I'm unable to find what causes this, WordWrap is set to false, and ellipsis is not specifically set or drawn

**Comment by RandallFlagg** [comment]
> somehow I'm unable to find what causes this, WordWrap is set to false, and ellipsis is not specifically set or drawn

@Hirogen does this happen to you in the new version? If ye, can you exaplin how to reproduce? If not, maybe it is because the component has changed?

**Comment by Hirogen** [comment]
> > somehow I'm unable to find what causes this, WordWrap is set to false, and ellipsis is not specifically set or drawn
> 
> [@Hirogen](https://github.com/Hirogen) does this happen to you in the new version? If ye, can you exaplin how to reproduce? If not, maybe it is because the component has changed?

yes it happens, just filter a window with a line that is longer then 4k characters and then copy that line from the filter window, it will add [...] at the end, I could not find the property that is responsible

**Comment by Hirogen** [comment]
this is finally fixed... displayed lines can now be configured (20k is default), copying a line will no longer be truncated see #491
